src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-Encryption-CMK.AzureChinaCloud.json

```diff
@@ -158,9 +158,10 @@
       },
       "SqlServerTDECMKEffect": {
         "type": "String",
-        "defaultValue": "AuditIfNotExists",
+        "defaultValue": "Audit",
```

Comment:
version not increased on line 11?

Reply:
My bad, missed this one, it is now fixed.